### PR TITLE
v1beta1 support for pipeline list cmd

### PR DESCRIPTION
**Pull request by pradeepitm12**
this adds v1beta1 support for pipeline list command
minor refactor to taskrun list

Signed-off-by: Pradeep Kumar pradkuma@redhat.com

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [x] Run the code checkers with `make check`
- [x] Regenerate the manpages, docs and go formatting with `make generated`
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

```
release-note
```

### CODE REVIEW

**Comment by danielhelfand** [comment]
I'm wondering if https://github.com/tektoncd/pipeline/pull/2186 is related to this failure? The errors look very similar to when we tried to release using the go test Task from the catalog that had been updated to v1.14.

cc @vdemeester

**Comment by danielhelfand** [comment]
Opened #833, which should resolve the unit test issue. Seems to be more of an issue with format of the error from the http client.

**Comment by danielhelfand** [comment]
/test pull-tekton-cli-unit-tests

I think the test in question is flaky.

**Comment by pradeepitm12** [comment]
@danielhelfand Thanks for the quick bug fix and :+1:

File: pkg/pipeline/pipeline.go [hunk 1]
```diff
@@ -37,3 +44,23 @@ func GetAllPipelineNames(p cli.Params) ([]string, error) {
 	}
 	return ret, nil
 }
+
+func List(c *cli.Clients, opts metav1.ListOptions, ns string) (*v1beta1.PipelineList, error) {
+
+	pipelineGroupResource := schema.GroupVersionResource{Group: "tekton.dev", Resource: "pipelines"}
+	unstructuredTR, err := plist.AllObjecs(pipelineGroupResource, c, ns, opts)
```
Comment (danielhelfand) [inline pkg/pipeline/pipeline.go]: Any reason for calling this `unstructuredTR`? Maybe change to `P` or `Ps` for pipelines?

Comment (pradeepitm12) [reply pkg/pipeline/pipeline.go]: Good catch it's my bad, copy-paste err

**Pull request merged**